app/controllers/concerns/helpeable.rb

```diff
@@ -0,0 +1,48 @@
+module Helpeable
+  extend ActiveSupport::Concern
+
+  require 'active_support/core_ext/object/try'
+  require 'action_controller'
+
+  included do
+
+    def self.permit_recursive_params(params)
+      Rails.logger.info "ccjhchch"
+      # Rails.logger.info params.inspect
+      (params.try(:to_unsafe_h) || params).map do |key, value|
```

Comment:
What are you trying to do here? It looks overly complicated.

Reply:
this has been changed as well  in 0623cd0